internal/util/logging.go

```diff
@@ -68,7 +68,7 @@ func getZapLevel(level string) (zapcore.Level, error) {
 	return res, nil
 }
 
-func getZapEncoding(typ string) (zapcore.Encoder, error) {
+func GetZapEncoding(typ string) (zapcore.Encoder, error) {
```

Comment:
```suggestion
func GetZapEncoding(type string) (zapcore.Encoder, error) {
```
I think one letter is not worth abbreviation

Reply:
`type` is one of very few keywords in Go and can't be used as an identifier: https://go.dev/ref/spec#Keywords

Reply:
Right, nevermind